webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -239,6 +244,12 @@ def bugzilla_submit_url(report, bug_product):
     elif op_sys in ('Windows Unknown', 'Windows 2000'):
         op_sys = 'Windows'
 
+    comment = render_to_string('crashstats/bugzilla_comment.txt', {
+        'uuid': report['uuid'],
+        'parsed_dump': parsed_dump,
+        'crashing_thread': crashing_thread,
+    })
```

Comment:
I think I would calculate the stack portion in Python-land rather than in Jinja2 land and then pass it in as a rendering param. I'd much rather people be writing PRs to tweak the stack rendering code in python-land than in Jinja2 land.